### ISSUE TITLE
shiny app?  

### ISSUE THREAD

**Issue by wcornwell** [tmesaglio/Australian-plant-photos#17]
This could be a shiny app where you select an area and it tells you which unphotographed species to look for?

**Comment by wcornwell** [comment]
this can be done at a late stage of the paper, maybe even after acceptance--could be something that RBG might be into? @eflowerproject ?

**Comment by tmesaglio** [comment]
sounds like a cool idea to me

**Comment by eflowerproject** [comment]
yes, i agree cool idea for after submission; i don't think RBG is modern enough to understand the benefit of hosting such a tool though :-(
